### PR TITLE
codeine-246

### PR DESCRIPTION
**Pull request by ezraroi**
This change is [Reviewable](https://reviewable.io/reviews/codeine-cd/codeine/248)

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -7,6 +7,8 @@
 	<name>codeine</name>
 	<properties>
 		<prometheus_client_version>0.3.0</prometheus_client_version>
+		<drop_wizard_version>3.2.6</drop_wizard_version>
+		<jetty_version>9.4.8.v20171121</jetty_version>
```
Comment (lchayoun) [inline pom.xml]: Fix conflict with master

File: src/common/codeine/AbstractCodeineBootstrap.java [hunk 2]
```diff
@@ -131,6 +139,11 @@ private void registerPrometheus() {
         new JettyStatisticsCollector(stats).register();
     }
 
+    private void addHealthEndpoint(ServletContextHandler handler, HealthCheckRegistry healthCheckRegistry) {
+        log.info("Adding health end point");
+//        handler.addServlet(new ServletHolder(new HealthServlet(healthCheckRegistry)), Constants.HEALTH_CONTEXT);
```
Comment (lchayoun) [inline src/common/codeine/AbstractCodeineBootstrap.java]: ?

File: src/common/codeine/ConsulRegistrator.java [hunk 3]
```diff
@@ -35,12 +36,8 @@ public boolean consulExists() {
...
         log.info("Will register " + name + " with port " + port);
         AgentClient agentClient = client.agentClient();
-        Registration service = ImmutableRegistration.builder()
-            .id(name)
-            .name(name)
-            .port(port)
-            .check(Registration.RegCheck.tcp("localhost:" + port, 10, 5))
-            .build();
+        Registration service = ImmutableRegistration.builder().id(name).name(name).port(port)
+            .check(Registration.RegCheck.http("localhost:" + port + "/" + Constants.HEALTH_CONTEXT, 60, 5)).build();
```
Comment (lchayoun) [inline src/common/codeine/ConsulRegistrator.java]: All of the other components in our environment are using the ip for registration and health checks

File: src/common/codeine/AbstractCodeineBootstrap.java (hunk 2 above)
Comment (ezraroi) [reply src/common/codeine/AbstractCodeineBootstrap.java]: yes, this not needed.. forgot to delete it

File: src/common/codeine/ConsulRegistrator.java (hunk 3 above)
Comment (ezraroi) [reply src/common/codeine/ConsulRegistrator.java]: i am not sure i need to use the ip address for health checks registration. The peer is registering with the IP already

**Pull request merged**